kernel/Kconfig

```diff
@@ -792,6 +792,14 @@ config SMP
 	  When true, kernel will be built with SMP support, allowing
 	  more than one CPU to schedule Zephyr tasks at a time.
 
+config SMP_BOOT_DELAY
```

Comment:
Still don't love this, but it's basically harmless.  We can come in and refactor the way SMP CPU bringup happens later I guess.